btr-web/btr-main-app/services/file-significant-individual.ts

```diff
@@ -38,6 +38,7 @@ const getCurrentOwners = async (businessIdentifier: string) => {
     for (const si of data.value) {
       si.percentOfShares = si.percentOfShares ? si.percentOfShares.toString() : '0'
       si.percentOfVotes = si.percentOfVotes ? si.percentOfVotes.toString() : '0'
+      si.profile.citizenshipsExCA = si.profile.citizenshipsExCA || []
```

Comment:
Nit. not breaking :)
Is this exceptCA, right ? When I first see Ex, I think like previous Ca citizenship.

Reply:
Yeah, no I know, thats why I added nit, and approved.
Additionally if you are more used to it we should not even change it, as its either more clear for me or for you ;-)